### ISSUE TITLE
FastHttpLocust times out when HttpLocust does not

### ISSUE THREAD

**Issue by ThatOhio** [locustio/locust#1246]
### Describe the bug
While running some performance tests the last few weeks, I noticed that while using the FastHttpLocust client my requests would timeout at 5 seconds consistently on any request that took over 5 seconds to return. When using HttpLocust there was no timeout. I did not see an obvious setting I could change to increase the timeout for FastHttpLocust, and even if that setting exists, I would expect the defaults for both to be the same. 

### Expected behavior
The length of the timeout for both FastHttpLocust and HttpLocust should be the same, so that a timeout does not occur while using one or the other.

### Actual behavior
If a request made with FastHttpLocust takes more than 5 seconds, but less than the default timeout for HttpLocust (I do not know this timeout, I believe it is 60 seconds?), the request should not timeout. The default timeout settings for FastHttpLocust should mirror that of HttpLocust.

### Steps to reproduce
We setup an endpoint that simply waited 10 seconds and then returned, running the same locust file but swapping out HttpLocust with FastHttpLocust would produce different results (FastHttpLocust timing out after 5 seconds where HttpLocust did not). 

### Environment

- OS: Ubuntu
- Python version: 3.6.9
- Locust version: 0.13.5
- Locust command line that you ran: locust --host=http://web.site -f some_file.py
- Locust file contents (anonymized if necessary): 
```
from locust import HttpLocust, TaskSet, between
from locust.contrib.fasthttp import FastHttpLocust


def TimeoutTest(l):
    l.client.get('/timeout_test.php')


class UserBehavior(TaskSet):
    tasks = {TimeoutTest: 1}


class WebsiteUser(FastHttpLocust):
    task_set = UserBehavior
    wait_time = between(1.0, 2.0)
```
Swapping FastHttpLocust for HttpLocust to perform the same request with the HttpLocust. Our "/timeout_test.php" page was simply waiting 10 seconds to return.

**Comment by heyman** [comment]
Yes, gevenhttpclient (which is what FastHttpLocust uses) has a default timeout of 5 seconds. We should provide a way of specifying the timeout.

**Comment by cyberw** [comment]
I upped the timeout to 60 seconds because I encountered this problem myself (in master, not yet part of a release)

**Comment by ThatOhio** [comment]
Awesome, I'll pull down the master branch when I need to extend the timeout until the next release, thanks!

**Comment by cyberw** [comment]
Released with 0.14.0!
